samples/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(NOVELRT_SAMPLES_HEADERS)
```

Comment:
The samples project just includes `main.cpp`, references `libNovelRT`, and copies over its output.